src/main/java/org/jabref/logic/importer/fileformat/EndnoteXmlImporter.java

```diff
@@ -339,3 +502,8 @@ public List<BibEntry> parseEntries(InputStream inputStream) {
         return Collections.emptyList();
     }
 }
+
```

Comment:
These are too many empty llines at the end. OK for me now to keep things going.

Reply:
Ok, will pay attention to this next time as well.